spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`.
+
+In the case of a backed enum, there MUST NOT be a space between the enum name and colon, and there MUST be exactly one
+space between the colon and the backing type.
```

Comment:
It might be good to add a nod to return type declaration requirements since these are the same.